src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  sourceString.replace(/;\s*;/g, ' ').trim();
```

Comment:
The regular expression used in the `replace` method is not correct for the intended purpose. It replaces a semicolon followed by any amount of whitespace and another semicolon with a single space, which may not be the desired effect. If the goal is to remove empty declarations, you should replace them with an empty string instead.